src/modules/housingUnitType/dto/HousingUnitTypeCreate.dto.ts

```diff
@@ -11,55 +12,72 @@ import {
 import { HousingUnitTypeMediaCreateDTO } from './HousingUnitTypeMediaCreate.dto'
 
 export class HousingUnitTypeCreateDTO {
+    @ApiProperty({ example: 'Deluxe' })
     @IsDefined()
     @IsString()
     name!: string
 
+    @ApiProperty({ example: 'deluxe-suite' })
     @IsDefined()
     @IsString()
     slug!: string
 
+    @ApiProperty({ example: 'a deluxe suite for 2 people' })
```

Comment:
```suggestion
    @ApiProperty({ example: 'a deluxe suite for 2 people', required: false })
```

campo não é obrigatório